File: rabbit_mq/src/main/java/com/rabbit/rabbit_mq/domain/chat/entity/ChatMessage.java
package com.rabbit.rabbit_mq.domain.chat.entity;


import com.rabbit.rabbit_mq.domain.member.entity.Member;
import com.rabbit.rabbit_mq.global.jpa.BaseTime;
import jakarta.persistence.Entity;
import jakarta.persistence.ManyToOne;
import lombok.*;

import static lombok.AccessLevel.PROTECTED;

@Entity
@Builder
@NoArgsConstructor(access = PROTECTED)
@AllArgsConstructor(access = PROTECTED)
@Setter
@Getter
public class ChatMessage extends BaseTime {
    @ManyToOne
    private ChatRoom chatRoom;
    @ManyToOne
    private Member writer;
    private String body;
}

File: k_rstudy/src/main/kotlin/com/krstudy/kapi/domain/chat/entity/ChatMessage.kt
package com.krstudy.kapi.domain.chat.entity

import com.krstudy.kapi.global.jpa.BaseEntity

data class ChatMessage(
    var chatRoomId: Long = 0,
    var writerName: String? = null,
    var content: String? = null
):BaseEntity(){}